src/utils.js

```diff
@@ -351,3 +348,14 @@ export const omitKeys = obj => {
     return newObj;
   }, {});
 };
+
+export const safeExecute = (
```

Comment:
support async to

Reply:
There is still no async function that needs `safeExecute`. We can add async support if we'll need it in the future :)